### PR TITLE
refactor(core): clean up static files

### PR DESCRIPTION
**Pull request by koeaw**
Move CSS for Material Symbols to appropriate file, add, clean up comments, formatting in CSS files.

### CODE REVIEW

**Comment by koeaw** [comment]
Clean up to base work on fixes for UI annoyances with multi-select dropdowns for Frischmuth on (for now; can then be transferred to here if deemed useful).

**Review by richardhadden — APPROVED** [review]
Don't see any problem with merging this. I can't find a mention of `.material-symbols-align` CSS class anywhere in the code, so guess it's not used

File: apis_core/core/static/css/core.css [hunk 1]
```diff
@@ -1,11 +1,7 @@
+/* ACDH-CH logo in site footer */
```
Comment (b1rger) [inline apis_core/core/static/css/core.css:1]:
```suggestion
/* Logo in site footer */
```

**Review by b1rger — CHANGES_REQUESTED** [review]

**Comment by koeaw** [comment]
> Don't see any problem with merging this. I can't find a mention of `.material-symbols-align` CSS class anywhere in the code, so guess it's not used

It's used in a bunch of templates, e.g. in `apis_entities` `partials`.

File: apis_core/core/static/css/core.css (hunk 1 above)
Comment (koeaw) [reply apis_core/core/static/css/core.css]: Thanks, I hadn't considered it's meant to be a place holder. D'oh.

I'd actually like to make that clearer  – i.e. change it to sth. like "project logo", "app logo", "site logo". What do you think?

Comment (koeaw) [reply apis_core/core/static/css/core.css]: I've updated the wording. Can remove it again, but think it doesn't hurt to be specific about it.

**Review by b1rger — APPROVED** [review]

**Pull request merged**